src/ontology/config/mondo-match-rules.yaml

```diff
@@ -88,26 +88,26 @@ rules:
   
   - synonymizer:
       description: Broad match terms with the term 'other' in them.
-      match: r'(?i)^Other '
+      match: '(?i)^Other '
       match_scope: "*"
       replacement: ""
       qualifier: "broad"
 
   - synonymizer:
       description: Replace disease with disorder in the label.
-      match: r'disease'
+      match: 'disease'
       match_scope: "*"
       replacement: "disorder"
 
   - synonymizer:
       description: Replace numbers by "type number" in the label.
-      match: r'(?<!type)\s(\d)'
+      match: '(?<!type)\s(\d)'
       match_scope: "*"
       replacement: " type \\1"
 
   - synonymizer:
       description: Replace "'s" by "s" in the label.
-      match: r'\'s'
+      match: "[']s"
```

Comment:
I'm unfamiliar with these rules and these updates. Just going to trust you guys on this one!

Reply:
See the comments at the top of the PR from Nico which explains it all. `r` is python syntax, not yaml! https://stackoverflow.com/questions/33729045/what-does-an-r-represent-before-a-string-in-python

This used to work because the code would pass the yaml string through a python `eval`. I hope it's clear why that is wrong!

Reply:
This is each to check:

z.yaml
```yaml
foo: '\[[^)]*\]'
```

```python
>>> import yaml
>>> obj = yaml.safe_load(open("z.yaml"))
>>> obj
{'foo': '\\[[^)]*\\]'}
>>> import re
>>> re.compile(obj["foo"])
re.compile('\\[[^)]*\\]')
>>> pattern=re.compile(obj["foo"])
>>> pattern.match('[x]')
<re.Match object; span=(0, 3), match='[x]'>
>>> pattern.match('[)]')
>>> pattern.match('a')
>>> pattern.match('[]')
<re.Match object; span=(0, 2), match='[]'>
>>> pattern.match('[xx')
>>>
```

so works as presumably expected

Reply:
Oh, thanks for taking the time! That is indeed what I was more or less what suspecting regarding the regex / `r'` changes.